views/survey.pug

```diff
@@ -65,5 +65,7 @@ html
             input(type='hidden' name=name value=value)
           if start_time
             input(type="hidden" name='start_time' value=start_time)
+          if worker_id
+            input(type="hidden" name = "WorkerId", value= worker_id)

```

Comment:
Weren't we already getting this for free with the query loop a few lines up? (it should put anything in the query into the resulting form submission)

Reply:
Yup that is a bug.